src/source/reporting-examples.rst

```diff
@@ -0,0 +1,248 @@
+===============   ==========   ===================
+    1234           200          400
+    5678           300          400
+===============   ==========   ===================
+
+
+Example 3:
+************************************************************************
+
+Get all clicks and impressions for categories ``1234`` and ``5678`` for the past week, but split performance metrics per day and category. In addition, sort by date in ascending direction:
+
+.. code:: javascript
+
+
+    POST /api/sellside/metrics/data
+    Content-Type: application/sellside.metrics.data-v1+json
+    Accept: application/json
+    {
+        "timeRanges": [{
+            "period": "lastWeek"
+        }],
+        "dimensions": ["am:date", "am:categoryID"],
+        "metrics": ["am:clicks", "am:impressions"],
+        "filters": [{
+            "field": "am:categoryID",
+            "operator": "in",
+            "value": [1234, 5678]
+        }],
+        "sorts":[
+        {
+            "field":"am:date",
+            "direction":"asc"
+        }],
+    }
+
+
+.. code:: javascript
+
+    {
+        "data": [{
+            "rows": [{
+                "dimensions": ["2018-12-08 00:00:00", "1234"],
+                "metrics": [11, 12]
+            },
+            {
+                "dimensions": ["2018-12-08 00:00:00", "5678"],
+                "metrics": [9, 20]
+            },
+            {
+                "dimensions": ["2018-12-09 00:00:00", "1234"],
+                "metrics": [34, 67]
+            },
+                        {
+                "dimensions": ["2018-12-09 00:00:00", "5678"],
+                "metrics": [19, 20]
+            },
+            ...
+            {
+                "dimensions": ["2018-12-14 00:00:00", "1234"],
+                "metrics": [12, 90]
+            },
+            {
+                "dimensions": ["2018-12-14 00:00:00", "5678"],
+                "metrics": [43, 76]
+            }],
+        "count": 54            
+        }]
+    }
+
+
+=====================   ===============   ==========   ===================
+  am:date                 am:categoryID    am:clicks     am:impressions
+=====================   ===============   ==========   ===================
+ 2018-12-08 00:00:00        1234           11           12
+ 2018-12-08 00:00:00        5678           9            20
+ 2018-12-09 00:00:00        1234           34           67
+ 2018-12-09 00:00:00        5678           19           20
+ ...
+ 2018-12-14 00:00:00        1234           12           90
+ 2018-12-14 00:00:00        5678           43           76 
+=====================   ===============   ==========   ===================
+
+
+Example 4:
+************************************************************************
+
+Get all clicks, and average CPC for categories ``1234`` and ``5678`` for the past week, but split performance metrics per ad ID. In addition, enrich the response rows with current ad title and vendorID. Limit to 3 results:
```

Comment:
past week = previous week. Perhaps we need to document somewhere that weeks run from monday to sunday?